core/object/callable_method_pointer.cpp

```diff
@@ -90,4 +98,11 @@ void CallableCustomMethodPointerBase::_setup(uint32_t *p_base_ptr, uint32_t p_pt
 			h = hash_murmur3_one_32(comp_ptr[i], h);
 		}
 	}
+
+#ifdef DEBUG_ENABLED
```

Comment:
If this is only meant for developers, this should use `DEV_ENABLED`.

Reply:
Fixed